user-management-react-app/src/actions/index.js

```diff
@@ -0,0 +1,15 @@
+export const submitForm = (values) => {
+  return {
+    type: 'SUBMIT_FORM',
```

Comment:
Change action name to something more meaningful name. As in future you may have multiple forms at that time SUBMIT_FORM will be confusing.